### PR TITLE
Dev server negotiation

### PR DESCRIPTION
**Pull request by Cobus-Redelinghuys**
Merged negotiation-unit tests and added update front end requested.

### CODE REVIEW

**Comment by Cobus-Redelinghuys** [comment]
If possible when merging dont take the new application.properties. Take the original one. If not possible please contact me and I will provide the correct one

File: Smart-Contract-Verifier-Server/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-spring.datasource.url=jdbc:postgresql://db-postgresql-nyc3-98163-do-user-8880908-0.b.db.ondigitalocean.com:25060/verifierserver?sslmode=require
-spring.datasource.username=${dbuser}
-spring.datasource.password=${dbpassword}
+spring.datasource.url=jdbc:postgresql://localhost:5432/verifierserver
+spring.datasource.username=postgres
+spring.datasource.password=smartcontractserverdev
```
Comment (Kaleb-Bruwer) [inline Smart-Contract-Verifier-Server/src/main/resources/application.properties]: No

**Review by Kaleb-Bruwer — CHANGES_REQUESTED** [review]
Please remove login info from application.properties

File: Smart-Contract-Verifier-Server/src/main/resources/application.properties (hunk 1 above)
Comment (Cobus-Redelinghuys) [reply Smart-Contract-Verifier-Server/src/main/resources/application.properties]: I pushed a new application.properties. This was the original one that was in this branch.

**Review by Kaleb-Bruwer — APPROVED** [review]

**Pull request merged**